pkg/client/session_id_injector_interceptor.go

```diff
@@ -30,7 +30,7 @@ func (c *immuClient) SessionIDInjectorInterceptor(ctx context.Context, method st
 	return ris
 }
 
-// ClientStreamInterceptor gRPC client interceptor for streams
+// SessionIDInjectorInterceptor gRPC client interceptor for streams
```

Comment:
SessionIDInjectorInterceptor is a gRPC client interceptor for streams may be?

Reply:
Fixed in 41732fdc